openpgp/packet/config.go

```diff
@@ -52,6 +52,9 @@ type Config struct {
 	Algorithm PublicKeyAlgorithm
 	// Some known primes that are optionally prepopulated by the caller
 	RSAPrimes []*big.Int
+	// Curve configures the desired packet.Curve if the Algorithm is
+	// PubKeyAlgoECDSA or PubKeyAlgoEdDSA. If empty Curve25519 is used.
```

Comment:
This config can also be passed when adding a subkey, so it also applies when Algorithm is PubKeyAlgoECDH